src/content/docs/synthetics/synthetic-monitoring/private-locations/job-manager-transition-guide.mdx

```diff
@@ -9,184 +9,199 @@ redirects:
 freshnessValidatedDate: never
 ---
 
-In early 2022, we released a [new runtime](/docs/synthetics/synthetic-monitoring/using-monitors/new-runtime) for public locations that simplified script creation, gave scripted monitors access to Chrome 100 and Node.js 16, and introduced automated monitor management with our [NerdGraph API](/docs/apis/nerdgraph/examples/nerdgraph-synthetics-tutorial). 
-
-The synthetics job manager brings monitors on the newest runtime to your private locations, enabling your monitors to more closely match a real user's experience.
-
-If you're ready to install the job manager, you can check out our [install doc](/docs/synthetics/synthetic-monitoring/private-locations/install-job-manager) for installation procedures, or read through our [configuration doc](/docs/synthetics/synthetic-monitoring/private-locations/job-manager-configuration) for working with new or existing [private locations](/docs/synthetics/synthetic-monitoring/private-locations/private-locations-overview-monitor-internal-sites-add-new-locations). 
-
 <Callout variant="important">
-  On October 22, 2024, we will [end of life](/whats-new/2024/04/whats-new-04-09-eol-synthetics-runtime-cpm) the containerized private minion (CPM) and the legacy synthetics runtime versions it supports. Please review our [recommended migration steps](/docs/synthetics/synthetic-monitoring/private-locations/job-manager-transition-guide/#monitorMigration) to avoid degradation of your private location monitors. 
+    On October 22, 2024, we will [end of life](/whats-new/2024/04/whats-new-04-09-eol-synthetics-runtime-cpm) the containerized private minion (CPM) and the legacy synthetics runtime versions it supports. Please review our [recommended migration steps](/docs/synthetics/synthetic-monitoring/private-locations/job-manager-transition-guide/#monitorMigration) to avoid degradation of your private location monitors. 
 </Callout>
 
-## Smaller container images for your synthetics [#docker]
-
-Job managers execute your synthetics from runtime-specific container images tailored to the jobs you want to execute. While CPMs bundled legacy runtimes and artifacts into one growing file, our job manager instead separates our Chrome, Node, and job management work into separate, smaller images.
+The synthetics job manager brings your synthetic monitors to [newest runtime](/docs/synthetics/synthetic-monitoring/using-monitors/new-runtime) for your private locations. The synthetics job manager will replace Containerized Private Minions (CPMs), and lets you monitor your private locations with simplified script creation, access to Chrome 100 and Node.js 16, and automated monitor management with our [NerdGraph API](/docs/apis/nerdgraph/examples/nerdgraph-synthetics-tutorial). 
```

Comment:
Can you briefly explain why they need to install the job manager AND upgrade? What does this process look like from start to finish?

Reply:
For example (making this up so probably inaccurate):

1. Install job managerrs
2. Get new license key 
3. create new private location, or use existing location
4. Upgrade CPMs to job managers? or Create a new job manager?

Reply:
The recommended process and order is covered in the monitorMigration section of the document.